instrumentation/opentelemetry-instrumentation-system-metrics/setup.cfg

```diff
@@ -40,8 +40,8 @@ package_dir=
     =src
 packages=find_namespace:
 install_requires =
-    opentelemetry-api ~= 1.10.0
-    opentelemetry-sdk ~= 1.10.0
+    opentelemetry-api == 1.10.0
+    opentelemetry-sdk == 1.10.0
```

Comment:
Why is this 1.10.0? `callbacks` change will be new in 1.11.0?

Reply:
Oops. Was left in from the release pr.